.yarnrc.yml

```diff
@@ -29,3 +29,15 @@ plugins:
     spec: '@yarnpkg/plugin-interactive-tools'
 
 yarnPath: .yarn/releases/yarn-3.5.0.cjs
+
+npmRegistryServer: http://npmscopes.site.gs.com
```

Comment:
You can remove these parts the `.yarnrc.yml` shouldn't need to be part of your commit and can be ignored 👍